parser/ast.go

```diff
@@ -511,25 +511,26 @@ type DDL struct {
 
 // DDL strings.
 const (
-	CreateStr          = "create"
-	AlterStr           = "alter"
-	DropStr            = "drop"
-	RenameStr          = "rename"
-	TruncateStr        = "truncate"
-	CreateVindexStr    = "create vindex"
-	AddColVindexStr    = "add vindex"
-	DropColVindexStr   = "drop vindex"
-	AddIndexStr        = "add index"
-	CreateIndexStr     = "create index"
-	AddPrimaryKeyStr   = "add primary key"
-	AddForeignKeyStr   = "add foreign key"
-	CreatePolicyStr    = "create policy"
-	CreateViewStr      = "create view"
-	CreateMatViewStr   = "create materialized view"
-	CreateTriggerStr   = "create trigger"
-	CreateTypeStr      = "create type"
-	CommentStr         = "comment"
-	CreateExtensionStr = "create extension"
+	CreateStr                 = "create"
+	AlterStr                  = "alter"
+	DropStr                   = "drop"
+	RenameStr                 = "rename"
+	TruncateStr               = "truncate"
+	CreateVindexStr           = "create vindex"
+	AddColVindexStr           = "add vindex"
+	DropColVindexStr          = "drop vindex"
+	AddIndexStr               = "add index"
+	CreateIndexStr            = "create index"
+	AddPrimaryKeyStr          = "add primary key"
+	AddForeignKeyStr          = "add foreign key"
+	CreatePolicyStr           = "create policy"
+	CreateViewStr             = "create view"
+	CreateMatViewStr          = "create materialized view"
+	CreateSecurityTypeViewStr = "create sql security"
```

Comment:
他のものに合わせたほうがいいかな、と思いました。

```suggestion
	CreateSqlSecurityStr = "create sql security"
```